receiver/kubeletstatsreceiver/internal/kubelet/network.go

```diff
@@ -13,55 +13,27 @@
 // limitations under the License.
 
 package kubelet // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kubeletstatsreceiver/internal/kubelet"
-
```

Comment:
Is this an IDE formatting configured this way? :)

Reply:
Ya, at some point I had this whole file commented out, and when I started uncommenting things the IDE automattically re-added the import statements and did care about our newline lol. I'll look through the component and clean them up.

Reply:
Fixed